lib/templates/html/_sidebar_for_library.html

```diff
@@ -8,7 +8,7 @@
   {{/hasPublicClasses}}
 
   {{#hasPublicExtensions}}
-  <li class="section-title"><a href="{{{href}}}#extension">Extensions</a></li>
+  <li class="section-title"><a href="{{{href}}}#extensions">Extensions</a></li>
```

Comment:
Thanks, @geisterfurz007!   Rebuilt the aot renderer using `grind build` so this can land.